Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec_description.md

```diff
@@ -0,0 +1,4 @@
+## BaseIntegration Help
```

Comment:
Isn't that supposed to be the file description, who explain to customer how to get the product credential and how to config the instance.?
If so, I have changed the title, but it is not very detailed because there is not much to say to config LIS instance.

Reply:
Usually we explain how to get the credentials, is there a specific place to get it? Sometimes customers sees the integration and decide to install and buy license "on the go". You want it to be clear to them as possible where in the product should they get the key from.